config/locales/en/shared/schools/mentors.yml

```diff
@@ -3,8 +3,8 @@ en:
     schools:
       mentors:
         can_not_remove:
-          title: You can not remove this mentor
+          title: You cannot delete this mentor
           description: |
-            %{mentor_name} cannot be removed from your organisation because they are assigned to placements.
+            %{mentor_name} must be removed from current placements before you can delete them.
 
-            To remove %{mentor_name} as a mentor you must remove them from any current placements.
+            They are currently assigned to: 
```

Comment:
Related to https://github.com/DFE-Digital/itt-mentor-services/pull/923/files#r1720025712 the "They are currently assigned to:" should be moved to it's own locale so it can be included conditionally.